Assets/Scripts/Managers/PlayerDeathHandler.cs

```diff
@@ -68,6 +68,7 @@ void Update()
                     {
                         destroyed.GetComponent<SpriteRenderer>().enabled = true;
                         destroyed.GetComponent<Collider2D>().enabled = true;
+                        destroyed.layer = LayerMask.NameToLayer("Default");
```

Comment:
Why is this line here? You shouldn't need to change the layer of anything during reset.